src/status_im/contexts/profile/edit/name/view.cljs

```diff
@@ -17,7 +16,7 @@
   (let [insets              (safe-area/get-insets)
         profile             (rf/sub [:profile/profile-with-image])
         customization-color (rf/sub [:profile/customization-color])
-        display-name        (profile.utils/displayed-name profile)
+        display-name        (:display-name profile)
```

Comment:
It seems like we only want to edit the display name now, so I changed this screen to use the `:display-name` of the profile instead of using `profile.utils/displayed-named` for choosing between the ENS names, display-name, and the preferred name, and etc.